### ISSUE TITLE
Add `coverage.py` as dependency to start measuring code coverage

### ISSUE THREAD

**Issue by joshuadavidthomas** [nicoddemus/pytest-rich#70]
It's going to be abysmal at first 😬.

**Comment by joshuadavidthomas** [comment]
I've always used `pytest-cov` without really thinking about it, but Anthony Sottile brings up some good points in a recent video on his channel: https://www.youtube.com/watch?v=sPgvHGkmd0U. Maybe just use plain old `coverage.py`?

**Comment by nicoddemus** [comment]
Yes, specially for pytest plugins, I would recommend to use `coverage` directly.

**Comment by eliegoudout** [comment]
> Anthony Sottile brings up some good points in a recent video on his channel: https://www.youtube.com/watch?v=sPgvHGkmd0U.

I've just watched it and I essentially find that his most important point is the tedious option passing. But `pyproject.toml` configuration makes it irrelevant imo.

**Comment by nicoddemus** [comment]
Oh this ticket is not about running coverage automatically for end users. This ticket is just about using `coverage` in pytest-rich's own CI.

**Comment by eliegoudout** [comment]
I thought it was about handling the compatibility with `pytest-cov`, mb!

**Comment by joshuadavidthomas** [comment]
Yeah, younger, less experienced me who opened this issue was idealistic and thought "we must have coverage!" But these days I'm not so sure, especially for a library like this. I'd be okay closing this as not planned.

**Comment by joshuadavidthomas** [comment]
Apparently I still have at least triage access, so I'll go ahead and close it myself.

@nicoddemus I'll leave it up to you if you want to remove my access or not. This project has always been one I wanted to circle back to help out, but these days I'm not sure how much bandwidth I have.

**Comment by joshuadavidthomas** [comment]
> Thanks @joshuadavidthomas, let me know if the situation changes. 👍 

If nothing else I can lob some PRs your way when I find some time, no need for me to have the level of access I did anymore. 😄
